config/default_settings/zh_CN.yml

```diff
@@ -0,0 +1,11 @@
+posting:
+  min_post_length: 8
+  min_first_post_length: 8
+  min_private_message_post_length: 4
+  min_topic_title_length: 5
+  uncategorized_description: 不需要分类的主题，或者不适用于任何分类的主题。
```

Comment:
It's a site setting. Basically, the answer is no. Uncategorized category uses a different approach. Since it's a site setting, it can be changed by admin at any time and they also should change this site setting.